src/script/page/AppMain.tsx

```diff
@@ -73,15 +73,23 @@ const AppContainer: FC<AppContainerProps> = ({root}) => {
     closeRightSidebar();
   };
 
+  // To be changed when design chooses a breakpoint, the conditional can be integrated to the ui-kit directly
+  const smBreakpoint = useMatchMedia('max-width: 620px');
+
+  const responsiveView = useAppMainState(state => state.responsiveView);
+  const showLeftSidebar = responsiveView.currentView == ViewType.LEFT_SIDEBAR;
```

Comment:
```suggestion
  const {currentView} = useAppMainState(state => state.responsiveView);
  const isLeftSidebarVisible = currentView == ViewType.LEFT_SIDEBAR;
```

For me showLeftSidebar means to open it :)